boot/makebootable.go

```diff
@@ -531,6 +532,9 @@ func makeRunnableSystem(model *asserts.Model, bootWith *BootableSet, sealer *Tru
 		if makeOpts.Standalone {
 			flags.SnapsDir = snapBlobDir
 		}
+
+		locker := makeOpts.Unlocker()
+		defer locker()
 		// seal the encryption key to the parameters specified in modeenv
 		if err := sealKeyToModeenv(sealer.dataEncryptionKey, sealer.saveEncryptionKey, model, modeenv, flags); err != nil {
```

Comment:
won't we hit the same issues with RunFDESetupHook as in the other PR? it just happens that we don't have tests that mix the API and the hook, because the API is used only on x86 but that is not a given

Reply:
Right. Maybe I should pass the unlocker down to the call to `sealKeyToModeenvUsingSecboot`.